### PR TITLE
Add "Retired" Rank

### PR DESCRIPTION
**Pull request by DeathlyPlays**
This is a semi-serious idea that I feel we should implement for the retired staff on SG, seeing as half of our Voices are "retired"

### CODE REVIEW

**Comment by DeathlyPlays** [comment]
@HoeenCoder thoughts?

File: config/config-example.js [hunk 1]
```diff
@@ -408,6 +408,14 @@ exports.grouplist = [
 		broadcast: true,
 	},
 	{
+		symbol: '♿',
+		id: "retired",
+		name: "Retired",
+		inherit: ' ',
```
Comment (ghost) [inline config/config-example.js]: `inherit: '+',` should do about the exact same thing

Comment (DeathlyPlays) [reply config/config-example.js]: The idea is to have it below the Voice rank seeing as it's essentially reg but with broadcast access

Comment (ghost) [reply config/config-example.js]: Fair Enough

Comment (HoeenCoder) [reply config/config-example.js]: Its should just inherit voice, it has the same perms (maybe more) anyways

**Comment by ghost** [comment]
Maybe something picky, but i feel like the retired auth should be above the voice

**Comment by DeathlyPlays** [comment]
Well technically they quit though @Gligar so they should be at the bottom I considered it too

**Comment by Mystifi** [comment]
I don't really like the Omega sign as the rank symbol, but it isn't the worst, I guess. Are there any other symbols we could debate using?

**Comment by ghost** [comment]
Here's some: $ ^ ? =

**Pull request closed without merging**